Formula/python@3.8.rb

```diff
@@ -1,8 +1,8 @@
 class PythonAT38 < Formula
```

Comment:
This needs to be changed to PythonAT381. I posted this comment before but it seems to have disappeared.